charter.md

```diff
@@ -53,15 +53,227 @@ The following responsibilities belong to the Steering Committe, but may be deleg
+recent election will vote in the special election i.e., eligibility will not be
+redetermined at the time of the special election. A committee member elected in
+a special election will serve out the remainder of the term for the person they
+are replacing, regardless of the length of that remainder.
+
+[voter-eligibility]: /elections.md#eligibility-for-voting
+
+### Resignation
+
+If a committee member chooses not to continue in their role, for whatever
+self-elected reason, they must notify the committee in writing.
+
+### Dissolution
+
+If committee members believe that the committee is no longer able to act in
+accordance with the above [mission](#mission), the committee may dissolve by a
+few methods:
+
+* Dissolution by vote
+* Dissolution by loss of committee
+
+See [elections](#elections) for special criteria in the instance of a general
+election triggered by dissolution.
+
+#### Continuity
+
+If the committee is dissolved completely by any mechanism, selected members
+(selection described below) agree to be "continuity advisors" and attend
+meetings of the subsequently elected incoming committee in a non-voting
+capacity for 6 meetings or until the incoming committee dismisses them,
+whichever is first.
+
+##### Selection
+
+At least two (2) and not more than three (3) members shall be selected for
+this duty.
+
+* One shall be the longest-serving outgoing committee member. If multiple, the
+  incoming committee may select between them.
+* The second shall be selected by the incoming committee at their discretion.
+* The incoming committee, at their discretion, may select a third.
+
+#### Dissolution by vote
+
+Dissolution requires an affirmative vote of a **_two-thirds supermajority of
+the [fixed membership of the committee](#composition)_**, and must be unanimous
+when the committee has four (4) or fewer members.
+
+Example:
+
+* 7 (members) / 3 = 2.333...
+* 2.333... * 2 = 4.666...
+* Round up to the nearest whole number (5)
+* An affirmative vote of 5 members would be required to dissolve the committee
+
+The committee must specify a date of dissolution.
+If the committee is dissolved, all seats are vacated on the date specified.
+
+#### Dissolution by loss of committee
+
+If at any time a **_majority of the
+[fixed membership of the committee](#composition)_**'s seats are vacant, the entire committee will be dissolved.
+
+Example:
+
+* 7 (seats) / 2 = 3.5
+* Round up to the nearest whole number (4)
+* If 4 committee seats are vacant, the entire committee will dissolve
+
+The number of vacant seats will be evaluated prior to any replacement of
+previously-seated members.
+
+### Removal
+
+#### No confidence
+
+A Steering Committee member may be removed by a three-fourths vote of no
+confidence from other Steering Committee members.
+
+The call for a vote of no confidence will happen in a public Steering Committee meeting and a
+GitHub issue in kubernetes/steering by a current member and must be seconded by
```

Comment:
Respectfully, there are cases where the specifics of a CoC violation cannot be revealed to protect a victim and/or reporter. The action taken can be public, but the actual specifics around the violation should not be revealed in public without careful consideration of the safety of folks involved.

Reply:
> Respectfully, there are cases where the specifics of a CoC violation cannot be revealed to protect a victim and/or reporter. The action taken can be public, but the actual specifics around the violation should not be revealed in public without careful consideration of the safety of folks involved.


Then it should be possible to point to the public censure against the steering member.  A CoC violation that does not result in a public censure of some kind seems like a thin pretext for removing the member from office.

Reply:
Ah, I misunderstood your point, then. I thought you were saying the specifics of the violation were needed for the community to judge, rather than saying there needed to be an acknowledgement that there was a violation and censure.

Reply:
@deads2k A censure is likely to work when the person censured is likely to change what they do, for those who are likely to continue the same response to a situation, how many censures are enough? Why put community through the harm again and again?

Reply:
I do think it would be worthwhile for the other (non-affected) members of Steering to produce a _short_, general statement providing some context for the vote of no confidence.

Personally, as a community member and voter, I'd want some level of explanation:

- to inform my future votes, and especially:
- in the absence of any restrictions from that member running in future elections (current state)

I do not however, think it would be useful to mandate:

- a format in which community feedback should be received by the committee
- that that feedback be committed to public fora

I feel that we as Steering members find ways to catalyze private feedback.

If that is a misaligned assumption on my part, then it's something that should discussed as part of https://github.com/kubernetes/steering/issues/250 (which will iterate on feedback on elections process and dissolution of committee).

Reply:
> I do not however, think it would be useful to mandate:
> 
> * a format in which community feedback should be received by the committee
> * that that feedback be committed to public fora

How will the community be able to inspect and understand why their elected representative was removed if we do not require that the reasons for the removal be public?

Reply:
> Personally, as a community member and voter, I'd want some level of explanation:

I think it's also important to see how each steering member voted.  I think it's possible that some community members would disagree with removing a steering member and would want to explicitly down-vote every standing steering member who voted for removal.

Reply:
Added providing some short statement to set context on the vote in https://github.com/kubernetes/steering/pull/248/commits/7564e40a553263a936b337201f687e45a0e7d356.